rosetta/scripts/params.go

```diff
@@ -12,25 +12,24 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-package model
+package scripts
 
 import (
-	"github.com/onflow/flow-go/ledger"
 	"github.com/onflow/flow-go/model/flow"
 )
 
-type State interface {
-	Last() (uint64, flow.StateCommitment)
-	Raw() Raw
-	Ledger() Ledger
+type Params struct {
+	FungibleToken flow.Address
 }
 
-type Raw interface {
-	WithHeight(height uint64) Raw
-	Get(key []byte) ([]byte, error)
+func TestNet() Params {
+	return Params{
+		FungibleToken: flow.HexToAddress("0x9a0766d93b6608b7"),
+	}
 }
 
-type Ledger interface {
-	WithVersion(version uint8) Ledger
-	Get(*ledger.Query) ([]ledger.Value, error)
+func MainNet() Params {
+	return Params{
+		FungibleToken: flow.HexToAddress("0xf233dcee88fe0abe"),
+	}
```

Comment:
Shouldn't those addresses be in private constants?

This would have two positive results IMO: It would make the code more explicit, and would make it possible to recompile the DPS with a different set of addresses without changing the code. I'm not sure yet if that would make sense for end-to-end tests but it's what comes to my mind at least with my limited perspective on the matter

Reply:
I want to refactor the whole approach, but no. The idea is to give consumers access to these kind of chain-specific parameters.